src/formatDate.js

```diff
@@ -51,6 +51,54 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const fromSeperator = fromFormat.pop();
+  const dateArray = date.split(fromSeperator);
+  let dateYear, dateMonth, dateDay;
+  const output = [];
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i] === 'YYYY' || fromFormat[i] === 'YY') {
+      dateYear = dateArray[i];
+    }
+
+    if (fromFormat[i] === 'MM') {
+      dateMonth = dateArray[i];
+    }
+
+    if (fromFormat[i] === 'DD') {
+      dateDay = dateArray[i];
+    }
+  }
+
+  const toSeperator = toFormat.pop();
```

Comment:
I think this line can be moved to the top where other variables are defined.

Reply:
Done.

Reply:
Great